task-definition.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
probably need to clean this up, remove your specific variables, and have a readme about how to use this